mypy/typeanal.py

```diff
@@ -291,6 +291,7 @@ def anal_var_defs(self, var_defs: List[TypeVarDef]) -> List[TypeVarDef]:
 
     def builtin_type(self, fully_qualified_name: str, args: List[Type] = None) -> Instance:
         node = self.lookup_fqn_func(fully_qualified_name)
+        assert isinstance(node.node, TypeInfo)
         info = cast(TypeInfo, node.node)
```

Comment:
You should be able to get rid of this cast (and could get rid of `info` entirely).